test/WebSites/FiltersWebSite/Controllers/CombineAuthorizeController.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using System.Linq;
+
+namespace FiltersWebSite
```

Comment:
Would it be possible to use `SecurityWebSite` for this? `FiltersWebSite` is basically a [superfund](https://en.wikipedia.org/wiki/Superfund) site as far as its usability. I don't want to put any new tests that we care about in this project if at all possible.